### ISSUE TITLE
Shortened Certs

### ISSUE THREAD

**Issue by workaccountforwork1** [PaulJerimy/SecCertRoadmapHTML#135]
Is it possible to address these two bullet points in your change notes a different way? This seems like a bad change in my opinion.

"Shortened Security+, SSCP, GSEC, Programming Languages, CASP+, CISSP, CISSP Concentrations, and GSE to cover 4 spaces in only GRC to make room for more certifications

Shortened GREM to fit with the new size of the GRC certifications"

Shortening the certs in your chart like this makes it seem like the focus of these certs is more narrow. I just had someone ask me about CISSP, so I showed him this chart as I told him about it. I told him it was a very broad certification, but the current version of your chart does not reflect this. I had to show an older version of the chart in order to explain how the cert compares to other ones.

He was trying to decide between starting studying for CISM or CISSP. The current version of the chart does a poor job of illustrating the focus of the shortened certs now. It now looks like CISM is pretty similar to CISSP in terms of the topics that are covered.

**Comment by peterverwolf** [comment]
I fully agree with this comment. CISSP has 8 domains and is very broad.

**Comment by smurfhunter** [comment]
As someone studying for the CISSP right now, this confused me a lot. The 2023 print explained it better.

I think adding more explanation for GRC as a whole (the acronym is thrown haphazardly into the chart with minimal explanation) would be helpful.

**Comment by sinecurelife** [comment]
Alright, I've gotten a lot of feedback saying the new way sucks. I'll change it back, but I also need to find a different way to fit this whole monster on a 1080p screen with a few certs taken up 20+ slots.

If it helps, my thought process is that I don't spread all the certifications across every domain they touch, why should those few be special? But the feedback I'm getting is that CISSP/CASP+/SEC+ ect ARE special.

**Comment by smurfhunter** [comment]
> Alright, I've gotten a lot of feedback saying the new way sucks. I'll change it back, but I also need to find a different way to fit this whole monster on a 1080p screen with a few certs taken up 20+ slots.

Honestly, it's easy to throw out that it sucks, but I do see why the change was done and I don't know how I would change the certifications to indicate they cover the whole set of content that they do.

Maybe hover over them to see how many domains they cover, or add an asterisk? I think to a degree they are special, but at the same time, specialized certifications like the CCNA or RHCE (random examples) demonstrate better understanding of their respective domains to me because the person spent more time focusing on those areas in depth. However, I would consider a CISSP to be an expert with some understanding across multiple domains. I'm honestly not sure how to best convey that on a chart like this.

> But the feedback I'm getting is that CISSP/CASP+/SEC+ ect ARE special.

I think there's a few reasons for this. Primarily, employers treat them like they are special and the breadth they cover is LARGE.

**Comment by R-yu-2** [comment]
This is just my personal opinion, but since this roadmap was created based on the CISSP domain, it's only natural that CISSP covers a wider range of subjects.
Also, according to CompTIA's official roadmap, SEC+ covers a wider range of subjects, but CASP+ (now SecurityX) does not cover as much as I thought it would.
[https://partners.comptia.org/docs/default-source/resources/it-certification-roadmap-8-5x11-print0c3c382c09d76fc19da8ff0a002c2329](https://partners.comptia.org/docs/default-source/resources/it-certification-roadmap-8-5x11-print0c3c382c09d76fc19da8ff0a002c2329)

**Comment by smurfhunter** [comment]
> This is just my personal opinion, but since this roadmap was created based on the CISSP domain, it's only natural that CISSP covers a wider range of subjects. Also, according to CompTIA's official roadmap, SEC+ covers a wider range of subjects, but CASP+ (now SecurityX) does not cover as much as I thought it would. https://partners.comptia.org/docs/default-source/resources/it-certification-roadmap-8-5x11-print0c3c382c09d76fc19da8ff0a002c2329

While there are only four domains for the CASP+, it does cover a wider breadth of content than you'd expect. While getting my CISSP, I started studying with a CASP+ book I had on hand - I'd say it covers about 75% of the content the CISSP does. That and no validation of work experience are its two biggest pitfalls, and why I don't think it will end up being worth as much as the CISSP in the industry despite trying to compete with it, but it does have a wide coverage of topics.

Essentially any topic you'd expect to find on the Sec+, the CASP+ and CISSP will go into much more detail on while also having new content outright.

**Comment by JBarrskog** [comment]
I agree with the issue for the certs, and namely CISSP which I was showing today. 

What about introducing a new shape in the chart? Something like this:

```txt
        - - - - - - - - -
- - -  |      CISSP     | - - - 
        - - - - - - - - -

```

Or something along those lines. It would only take up 1 slot for the name, and would indicate its width without taking up too much space. 

Just an idea
